lib/foreman_ansible_core/runner/ansible_runner.rb

```diff
@@ -118,6 +120,14 @@ def verbose?
         @verbosity_level.to_i.positive?
       end
 
+      def forks_defined?
+        @ansible_forks.to_i.positive?
+      end
+
+      def forks
+        '--forks ' + @ansible_forks.to_s
```

Comment:
Because of how arguments are processed, you either need to insert `--forks` and the number as two separate elements into the `command` array or replace `--forks ` with `--forks=`